spring-cloud-alibaba-nacos-config/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -3,7 +3,6 @@
     {
       "name": "spring.cloud.nacos.server-addr",
       "type": "java.lang.String",
-      "defaultValue": "localhost:8848",
```

Comment:
default server-addr is localhost:8848, why remove it?

Reply:
code was accidentally lost while merging, I'll recover it